python/cugraph/community/subgraph_extraction.py

```diff
@@ -52,11 +52,11 @@ def subgraph(G, vertices):
 
     null_check(vertices)
 
-    result_graph = Graph()
+    result_graph = DiGraph()
```

Comment:
Can you please point me to where DiGraph is implemented?

Reply:
https://github.com/Iroy30/cugraph/blob/763aa86c45b945cc59b1e34e02d37368d65a8b29/python/cugraph/structure/graph.py#L426